src/custom_nodes/image_transformation/image_transformation.cpp

```diff
@@ -15,20 +15,70 @@
 //*****************************************************************************
 #include <iostream>
 #include <map>
+#include <shared_mutex>
 #include <string>
 
 #include "../../custom_node_interface.h"
+#include "../common/custom_node_library_internal_manager.hpp"
 #include "../common/opencv_utils.hpp"
 #include "../common/utils.hpp"
 #include "opencv2/opencv.hpp"
 
-static constexpr const char* TENSOR_NAME = "image";
+using CustomNodeLibraryInternalManager = ovms::custom_nodes_common::CustomNodeLibraryInternalManager;
+using BuffersQueue = ovms::custom_nodes_common::BuffersQueue;
+
+static constexpr const char* INPUT_IMAGE_TENSOR_NAME = "image_in";
```

Comment:
Is it really desired to change example node library input/output name in this release? People might already use this example node in their configurations and this will break their set ups after rebuilding 2022.1. This, besides other API changes, introduces yet another non-back-compatible change to OVMS.
@dtrawins @atobiszei

Reply:
If the name change is really desired, we have example nodes overview in our documentation files. The change is not reflected in this doc:
https://github.com/openvinotoolkit/model_server/tree/develop/src/custom_nodes/image_transformation#custom-node-inputs

Reply:
This comment is not related to `image_transformation` node only, but any example node we currently have

Reply:
I did not take that into consideration. This change is not necessary and we should stick to the previous names.

Reply:
Input and output names should now be restored